bin/ci

```diff
@@ -68,15 +100,40 @@ function update_sources() {
 #
 # Usage: update
 function update() {
-    update_sources
+    help="Utility function that generates a circle configuration file with a workflow
+for each site and running only the necessary jobs"
+    ignore_changelog=false
+
+    print_usage() {
+    printf "Usage: ..."
+    }
+
+    while true; do
+        case "${1}" in
+            --ignore-changelog|ignore-changelog)
```

Comment:
-i for a one letter option?

Reply:
I hesitate because this shorcuted option did not mention changelog.

Reply:
well... it's just a letter :man_shrugging: could be anything in the alphabet?